### DISCUSSION TITLE
[V3] How to loading target point to $parent action paremeter?

### DISCUSSION THREAD

**Discussion by przemi** [livewire/livewire#6498] [1. Help]
I have button with action wire:click="$parent.nextStep" how to point loading target to reference that action?
wire:target="$parent.nextStep" doesn't effect, do You have any idea how to do that?

**Comment by david371571** [comment]
Same question here, I tried to deep-dive into the source of the wire:loading directive but was not able to find a working pattern.
I temporary solved it using alpine like this:

```html
<button  x-data="{loading: false}" 
@click="async () => { loading = true; await $wire.$parent.myAction(); loading = false; }" 
:disabled="loading"
>
     Test
</button>
```
